src/open_inwoner/components/templatetags/side_navigation.py

```diff
@@ -142,23 +145,106 @@ def _extract_counter(self, node: NavigationNode) -> int | None:
+        logger.debug("Current page qualified URL name: '%s'", current_qualified_name)
 
-        # Handle special case where route doesn't match CMS page URLs
+        # Handle special case of redirect routes, where the route of the node is
+        # actually a redirect to another page: we want to verify if we're on the
+        # redirect target, not the node path.
         try:
-            resolved = resolve(self.request.path)
-            if resolved.url_name == "contactmoment_list":
-                current = "contactmomenten" in node.get_absolute_url()
+            if redirect_url := node.attr.get("redirect_url", None):
+                logger.debug(
+                    "Node '%s' has redirect_url configured: '%s'",
+                    node,
+                    redirect_url,
+                )
+                try:
+                    resolved_redirect_url = resolve(redirect_url)
+                    redirect_qualified_name = self._get_qualified_url_name(
+                        resolved_redirect_url
+                    )
+                    logger.debug(
+                        "Redirect target qualified URL name: '%s'",
+                        redirect_qualified_name,
+                    )
+                    if current_qualified_name == redirect_qualified_name:
+                        logger.debug(
+                            "Node '%s' redirect target '%s' matches current page '%s':"
+                            " marking as active",
+                            node,
+                            redirect_qualified_name,
+                            current_qualified_name,
+                        )
+                        return True
+                    else:
+                        logger.debug(
+                            "Node '%s' redirect target '%s' does not match current "
+                            "page '%s'",
+                            node,
+                            redirect_qualified_name,
+                            current_qualified_name,
+                        )
+                except Resolver404:
+                    logger.debug(
+                        "Could not resolve redirect_url '%s' for node '%s'",
+                        redirect_url,
+                        node,
+                    )
+
+            node_absolute_url = node.get_absolute_url()
+            resolved_node_path = resolve(node_absolute_url)
+            node_qualified_name = self._get_qualified_url_name(resolved_node_path)
+            logger.debug(
+                "Node '%s' qualified URL name: '%s'", node, node_qualified_name
+            )
+
+            is_match = current_qualified_name == node_qualified_name
+
+            # For CMS pages (and other catch-all patterns), URL names match but paths differ.
+            # When URL names match, also verify the actual paths match.
+            if is_match and current_qualified_name == node_qualified_name:
```

Comment:
```suggestion
            if is_match:
```

second conjunct is redundant (l. 209: `is_match = current_qualified_name == node_qualified_name`)

Reply:
Bad rebase, thanks!